src/components/LessonDateDisplay/LessonDateDisplay.js

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import "./LessonDateDisplay.scss";
 
 function LessonDateDisplay(props) {
+  const { firebase } = props;
```

Comment:
instead of manually assinging firebase as props you can do `function LessonDateDisplay({ firebase })`

Reply:
ope dis component not even supposed to be here